tests/dbus/_misc.py

```diff
@@ -22,6 +22,9 @@
 import subprocess
 import time
 
+from stratisd_client_dbus import get_object
```

Comment:
The changes in this commit "Wait for service to become ready" seem like generic improvements and not really coupled to the rest of the commits in this PR. So, it might as well be separated into a separate PR, for faster merging.